includes/wikia/models/DesignSystemGlobalFooterModel.class.php

```diff
@@ -420,218 +584,126 @@ public function getData() {
 				]
 			];
 		}
-
-		return $data;
 	}
 
-	private function getLicenseData() {
-		$licenseText = WikiFactory::getVarByName( 'wgRightsText', $this->wikiId )->cv_value ?: $this->wg->RightsText;
-
-		return [
-			'type' => 'link-text',
-			'title' => [
-				'type' => 'text',
-				'value' => $licenseText
-			],
-			'href' => $this->getLicenseUrl()
-		];
-	}
-
-	private function getFandomOverview() {
```

Comment:
Probably we should adjust unittests or at least its manes to reflect this change.